dojo/tools/sonarqube_api/importer.py

```diff
@@ -75,7 +75,9 @@ def import_issues(self, test):
                 line = issue.get('line')
                 rule_id = issue['rule']
                 rule = client.get_rule(rule_id)
-                severity = self.convert_sonar_severity(rule['severity'])
+                #severity = self.convert_sonar_severity(rule['severity'])
```

Comment:
Please remove both comments. If someone is interested in the history of the change they can use GitHub to find out what has changed and why.

Reply:
Removing the comments